src/utils/msc_tree.cc

```diff
@@ -410,6 +411,7 @@ TreeNode *CPTAddElement(unsigned char *ipdata, unsigned int ip_bitmask, CPTTree
                 node->count++;
                 new_node = node;
                 node->netmasks = reinterpret_cast<unsigned char *>(malloc(node->count *  sizeof(unsigned char)));
+                memset(node->netmasks, 0, (node->count *  sizeof(unsigned char)));
```

Comment:
Here is another potential crash if we are out of memory.  Malloc above may return null and there is no check below for this condition.